index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
```

Comment:
Thank you for the correction! Creating a non-visual flow is new to me so I thought that it would be beneficial to add alt-text to all images. I will address this for all design-only images.